### ISSUE TITLE
Tuning display progress is wrong

### ISSUE THREAD

**Issue by xavan83** [ultralytics/ultralytics#23540]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Other, Hyperparameter Tuning

### Bug

During tuning the display of the progress is incorrect. Instead of replacing 0% by 1%, a new line is added under the previous one (I don't see that for training):
41/100      2.93G     0.0356      7.142    0.02101         48        640: 27% ━━━───────── 105/389 7.2it/s 15.0s<39.7s
     41/100      2.93G    0.03565      7.139      0.021         40        640: 27% ━━━───────── 106/389 7.4it/s 15.1s<38.4s
     41/100      2.93G    0.03567      7.139    0.02102         52        640: 28% ━━━───────── 107/389 7.2it/s 15.2s<39.0s
     41/100      2.93G    0.03568       7.14    0.02102         53        640: 28% ━━━───────── 108/389 7.0it/s 15.4s<40.3s

### Environment

Ultralytics 8.4.10 🚀 Python-3.13.11 torch-2.10.0+cu130 CUDA:0 (NVIDIA GeForce RTX 5050, 7708MiB)
Setup complete ✅ (8 CPUs, 30.1 GB RAM, 214.9/465.4 GB disk)

OS                     Linux-6.17.0-12-generic-x86_64-with-glibc2.42
Environment            Linux
Python                 3.13.11
Install                pip
Path                   /home/xavan/miniconda3/envs/yolo313/lib/python3.13/site-packages/ultralytics
RAM                    30.14 GB
Disk                   214.9/465.4 GB
CPU                    12th Gen Intel Core(TM) i3-12100
CPU count              8
GPU                    NVIDIA GeForce RTX 5050, 7708MiB
GPU count              1
CUDA                   13.0

numpy                  ✅ 2.3.5>=1.23.0
matplotlib             ✅ 3.10.8>=3.3.0
opencv-python          ✅ 4.13.0.90>=4.6.0
pillow                 ✅ 12.0.0>=7.1.2
pyyaml                 ✅ 6.0.3>=5.3.1
requests               ✅ 2.32.5>=2.23.0
scipy                  ✅ 1.17.0>=1.4.1
torch                  ✅ 2.10.0+cu130>=1.8.0
torch                  ✅ 2.10.0+cu130!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision            ✅ 0.25.0+cu130>=0.9.0
psutil                 ✅ 7.2.1>=5.8.0
polars                 ✅ 1.37.1>=0.20.0
ultralytics-thop       ✅ 2.0.18>=2.0.18

### Minimal Reproducible Example

import wandb
from ultralytics import YOLO
wandb.init(project="YOLO-Tuning", entity="xxxxxx")
model = YOLO("yolo26n.pt")

result_grid = model.tune(data="/home/xxxxx/train/cropped.yaml", use_ray=True,gpu_per_trial=1)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xavan83, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

This is an automated response 🤖—an Ultralytics engineer will also assist soon 👀

If this is a 🐛 Bug Report (the tuning progress output “stacking” lines instead of updating in-place), please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. You’ve shared a great start—please also include:
-Your exact command/entrypoint (CLI vs Python) and whether this happens in a plain terminal, VSCode terminal, tmux/screen, or Jupyter
-The full console output from the start of `model.tune(...)` until a few duplicated progress lines appear
-Whether the behavior changes with `use_ray=True/False` (still within the same MRE)
-Your `ultralytics` version after upgrading (below) ✅

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
This is because your terminal window is too small. If it doesn't fit the width, it will print new lines

**Comment by xavan83** [comment]
I don't think so, there is still plenty of spaces. If I remember well I have the same behavior during training a few month ago and a version upgrade changed that.

<img width="1680" height="476" alt="Image" src="https://github.com/user-attachments/assets/b40fd0db-fd21-428f-bec4-8b1c6cc729fe" />

**Comment by Y-T-G** [comment]
Does it occur with normal training or just tuning?

**Comment by xavan83** [comment]
only for tuning

**Comment by Y-T-G** [comment]
I think it's just ray tune causing issues with the terminal output. Not due to Ultralytics.
